CMakeLists.txt

```diff
@@ -44,6 +96,7 @@ add_llvm_library(CodeAnalyzer MODULE
   src/CalledMethods.cpp
   src/MustCall.cpp
   src/DataflowPass.cpp
+  #src/ScopeAnalyzer.cpp
```

Comment:
Why was this file removed? It's usually a good idea to:
* keep lists of files in build scripts in sync with what's in the actual repo (so, e.g., if you want to delete something both delete the source and delete it from the script.
* generally prefer actually deleting things to commenting them out. The version control system preserves the previous state, so there's no need to worry that it'll be lost.